lalrpop/Cargo.toml

```diff
@@ -15,23 +15,64 @@ exclude = ["build.rs"]
 [lib]
 doctest = false
 
-[dependencies]
-ascii-canvas = "2.0"
-atty = "0.2"
-bit-set = "0.5"
-diff = "0.1.12"
-docopt = "1"
-ena = "0.14"
-itertools = "0.9"
-regex = "1"
-regex-syntax = "0.6"
-petgraph = "0.5"
-serde = "1.0"
-serde_derive = "1.0"
-string_cache = "0.8"
-term = "0.5"
-unicode-xid = "0.2"
-sha2 = "0.8"
+[dependencies.ascii-canvas]
```

Comment:
You can write these as `ascii-canvas = { version = "2.0", default_features = false }` etc. Personally I find that easier to read (less bloated).